api/views/auth.py

```diff
@@ -0,0 +1,119 @@
+import json
+from api import app, db
+from flask import Blueprint, request
+from flask import jsonify
+import api.auth_utils as auth_utils
+import api.auth_tokens as auth_tokens
+from api.auth_tokens import token_required
+from api.utils import create_response, row_constructor
+from api.models import User
+
+
+mod = Blueprint('auth', __name__)
+
+SIGNUP_URL = '/signup'
+LOGIN_URL = '/login'
+LOGOUT_URL = '/logout'

```

Comment:
can we namespace all these under `/auth`

aka we would have `/auth/signup` etc